### PR TITLE
Disable GHA annotations from the script

### PR DESCRIPTION
**Pull request by thekad**
ansible-lint has automatic GHA detection that annotates source code on each PR. This feature is still in Beta, disabling for now and will reenable later

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -33,22 +33,30 @@ jobs:
...
-          grep '^-[^-]' diff.output | sed -e 's/^-/-ERROR: /' | tee -a ${GITHUB_STEP_SUMMARY}
+          grep '^-' diff.output | sed -e 's/^-/-ERROR: /' | tee -a ${GITHUB_STEP_SUMMARY}
           echo '```' >> ${GITHUB_STEP_SUMMARY}
-          if grep -q '^-[^-]' diff.output; then
+          if grep -q '^-' diff.output; then
              echo "> Fix regressions listed above" | tee -a ${GITHUB_STEP_SUMMARY}
-             exit 1
+             regressions="$(grep '^-' diff.output | wc -l | cut -f 1 -d ' ')"
+             improvements="$(grep '^+' diff.output | wc -l | cut -f 1 -d ' ')"
+             # if ratio of regressions > improvements then fail this
```
Comment (fredericlepied) [inline .github/workflows/pr.yml:56]: counting doesn't implement a quality bar as you can change the regressions without noticing.

**Review by fredericlepied — CHANGES_REQUESTED** [review]
Do not count in the quality bar code.

File: .github/workflows/pr.yml (hunk 1 above)
Comment (thekad) [reply .github/workflows/pr.yml:56]: without counting or having a ratio this is a boolean check, either it passes 100% or it doesn't. Are you OK with this outcome?

Comment (fredericlepied) [reply .github/workflows/pr.yml:56]: It should not fail more what was failing in the main branch.

**Comment by fredericlepied** [comment]
can you rebase your change as I think the diff is coming because your branch is outdated compared to main?

**Comment by fredericlepied** [comment]
In fact, that is not the case. I don't understand why it is different with no code change :-(

**Comment by thekad** [comment]
> In fact, that is not the case. I don't understand why it is different with no code change :-(

No idea either, I am updating from branch from the GH UI see if that makes a difference

**Comment by fredericlepied** [comment]
> > In fact, that is not the case. I don't understand why it is different with no code change :-(
> 
> No idea either, I am updating from branch from the GH UI see if that makes a difference

It was fixed in main with Tatiana. I think this change is not needed anymore.

**Comment by thekad** [comment]
> > > In fact, that is not the case. I don't understand why it is different with no code change :-(
> > 
> > 
> > No idea either, I am updating from branch from the GH UI see if that makes a difference
> 
> It was fixed in main with Tatiana. I think this change is not needed anymore.

I have updated the description to the PR to just disable GHA annotations for now

**Review by fredericlepied — APPROVED** [review]

**Pull request merged**